mongodbatlas/database_users.go

```diff
@@ -158,7 +182,8 @@ func (s *DatabaseUsersServiceOp) Update(ctx context.Context, groupID, username s
 	}
 
 	basePath := fmt.Sprintf(dbUsersBasePath, groupID)
-	path := fmt.Sprintf("%s/admin/%s", basePath, username)
+
+	path := fmt.Sprintf("%s/%s/%s", basePath, updateRequest.GetAuthDB(), username)
```

Comment:
[non-blocking] I'd rather solve this the same way we approached `Delete` where the `databaseName` param was added, we already broke this service before when we did that may as well break it again

Reply:
we talked this off-line we'll keep this like it is